### DISCUSSION TITLE
Can you add the function of 'generating long screenshots of dashboard pages'?

### DISCUSSION THREAD

**Discussion by 81354791** [apache/superset#24722] [Ideas]
Hello author, first of all, thank you very much for the convenience your work has brought me.

At present, I have encountered a problem: I hope to be able to automatically get all the text and charts on the specified dashboard page and generate them into a long screenshot, so that I can use a robot to automatically send the pictures to my superiors at work. May I ask how I can achieve this?

<img width="451" alt="image" src="https://github.com/apache/superset/assets/50453959/3682539c-d59b-498b-bd8a-644265b8c85a">

**Comment by jfrag1** [comment]
It requires some extra setup, but the Alerts & Reports feature may be what you're looking for: https://superset.apache.org/docs/installation/alerts-reports/#alerts-and-reports

**Comment by rusackas** [comment]
@81354791 let us know if that Alerts & Reports feature works for you... otherwise I'll move this to an Ideas thread on Github Discussions since it's more of a feature request (a good one!) than it is a bug.
